This internal patch is cherry picked from commit
3bdb51e947e0595d4a0514f3cf29b321cd6f9483 and addresses an issue where
the [profiler] options appears in the wrong section. This change will
appear in the initial Mitaka release but has not yet been back-ported
to Kilo.

--- glance-2015.1.2/glance/opts.py.~2~	2016-01-24 16:51:48.000000000 -0800
+++ glance-2015.1.2/glance/opts.py	2016-01-25 19:10:40.596155047 -0800
@@ -52,7 +52,6 @@ _api_opts = [
         glance.common.wsgi.bind_opts,
         glance.common.wsgi.eventlet_opts,
         glance.common.wsgi.socket_opts,
-        glance.common.wsgi.profiler_opts,
         glance.image_cache.drivers.sqlite.sqlite_opts,
         glance.image_cache.image_cache_opts,
         glance.notifier.notifier_opts,
@@ -67,6 +66,7 @@ _api_opts = [
      glance.async.taskflow_executor.taskflow_executor_opts),
     ('store_type_location_strategy',
      glance.common.location_strategy.store_type.store_type_opts),
+    ('profiler', glance.common.wsgi.profiler_opts),
     ('paste_deploy', glance.common.config.paste_deploy_opts)
 ]
 _registry_opts = [
@@ -76,6 +76,7 @@ _registry_opts = [
         glance.common.wsgi.bind_opts,
         glance.common.wsgi.socket_opts,
         glance.common.wsgi.eventlet_opts))),
+    ('profiler', glance.common.wsgi.profiler_opts),
     ('paste_deploy', glance.common.config.paste_deploy_opts)
 ]
 _scrubber_opts = [
